packages/deces-backend/Makefile

```
@@ -276,9 +231,9 @@ backend-dev-stop:
 
 backend-dev-test:
 	@echo Testing API parameters
-	@docker exec -i ${USE_TTY} ${APP}-development bash /deces-backend/tests/test_query_params.sh
+	@docker exec -i ${USE_TTY} ${APP_BACKEND}-development bash /deces-backend/tests/test_query_params.sh
 
-dev: network backend-dev-stop ${WIKIDATA_LINKS} ${COMMUNES_JSON} ${DISPOSABLE_MAIL} ${DB_JSON} ${PROOFS} ${JOBS} smtp backend-dev
+dev: network backend-dev-stop backend-dev
```

checkmake (0.2.2)

[warning] Target "dev" should be declared PHONY.

(phonydeclared)